package.json

```diff
@@ -165,6 +167,7 @@
     "sanitize-filename": "^1.6.3",
     "semver": "^7.3.2",
     "showdown": "^1.9.1",
+    "source-map-support": "^0.5.19",
```

Comment:
For vue I guess?

Reply:
Good observation, that was a left over from me playing around. Removed.